src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -30,14 +30,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <StartupHookSupport>false</StartupHookSupport>
   </PropertyGroup>
 
-  <!-- We disable built-in Com support for trimmed apps here so that the feature
-       switch can flow to the runtimeconfig.json. Built-in Com is disabled
+  <!-- We disable Windows Forms Trimming support for trimmed apps here so that the feature
+       switch can flow to the runtimeconfig.json. enableWindowsFormsTrimming is disabled
        by default since they may require assemblies, types or members that
        could be removed by the linker, causing a trimmed app to crash. -->
-  <PropertyGroup Condition="'$(BuiltInComSupport)' == '' And
+  <PropertyGroup Condition="'$(enableWindowsFormsTrimming)' == '' And
```

Comment:
MSBuild properties are always PascalCased - so it should be `EnableWindowsFormsTrimming`.